packages/kernel-5.10/config-bottlerocket-metal

```diff
@@ -7,32 +7,67 @@ CONFIG_SATA_AHCI=y
 CONFIG_ATA=y
 CONFIG_ATA_PIIX=y
 
-# Intel network support
-CONFIG_IGB=m
-CONFIG_IGBVF=m
-CONFIG_NET_VENDOR_INTEL=y
-CONFIG_IGB_HWMON=y
-CONFIG_E1000=m
-CONFIG_E1000E=m
-CONFIG_E1000E_HWTS=y
+# AMD network support
+CONFIG_NET_VENDOR_AMD=y
+CONFIG_AMD_XGBE=m
+# CONFIG_AMD_XGBE_DCB is not set
 
 # Broadcom network support
 CONFIG_NET_VENDOR_BROADCOM=y
 CONFIG_TIGON3_HWMON=y
 CONFIG_TIGON3=m
+CONFIG_BNX2X=m
+CONFIG_BNX2X_SRIOV=y
 CONFIG_BNXT=m
 
+# Chelsio network support
+CONFIG_NET_VENDOR_CHELSIO=y
+CONFIG_CHELSIO_T4=m
```

Comment:
In the full config diff, I see the Chelsio T1 and T3 drivers are deselected/they're not explicitly selected here. I'm not familiar with Chelsio's product catalogue and haven't looked deeper yet, but those also seem to be 10Gbit/s Ethernet NICs. Why are we not selecting them as well?

Reply:
So as far as I understand the Chelsio T1 and T3 devices are mid 2000s vintage and have been discontinued for quite a while. All later models (T4, T5, T6) are serviced by the T4 variant of the driver. While those older T1/T3 cards have 10G support I left them out for that reason.

I am happy to take these if we find a use case for them or as a precaution to be complete here.

What we might want to use as a guide here: RHEL seems to not include these, while openSUSE does. Happy to have a discussion on this.

Reply:
Thanks for explaining, this makes sense. I agree on leaving them out and would be very surprised to still find them in any data center then.